src/Tablebot/Plugins/Say.hs

```diff
@@ -0,0 +1,34 @@
+-- |
+-- Module      : Tablebot.Plugins.Say
+-- Description : A command that outputs its input.
+-- Copyright   : (c) Amelie WD 2021
+-- License     : MIT
+-- Maintainer  : tablebot@ameliewd.com
+-- Stability   : experimental
+-- Portability : POSIX
+--
+-- A command that outputs its input.
+module Tablebot.Plugins.Say (sayPlugin) where
+
+import Data.Text (Text, pack)
+import Tablebot.Plugin
+import Tablebot.Plugin.Discord (Message, sendMessageVoid)
+import Tablebot.Plugin.Parser (untilEnd)
+import Text.Megaparsec
+
+-- | @say@ outputs its input.
+say :: Command
+say = Command "say" saycomm
+  where
+    saycomm :: Parser (Message -> DatabaseDiscord ())
+    saycomm = do
+      input <- untilEnd <?> ""
```

Comment:
Correct me if I'm wrong but does this make the error message just be blank if it can't find any items? is the `<?>` needed?